cypress/integration/entitybrowser_spec.js

```diff
@@ -1,59 +1,88 @@
 import helpers from '../helpers/helpers'
 
+const getPkFromUrl = url => url.substring(url.lastIndexOf('/') + 1)
+
+const createUser = (firstname, lastname) => {
+  return cy.request({
+    url: `${Cypress.env('BACKEND_URL')}/nice2/rest/entities/User`,
+    method: 'POST',
+    json: true,
+    body: {model: 'User', key: null, paths: {firstname, lastname, relGender: {display: 'Weiblich', key: '2'}}}
```

Comment:
hardcoded key `2` could lead to problems someday

Reply:
Do you think the pk of a gender can change some days? I think anything else than hardcode would be an overkill.